tests/chainerx_tests/unit_tests/routines_tests/test_indexing.py

```diff
@@ -127,6 +133,21 @@ def forward_xp(self, inputs, xp):
         return y,
 
 
+@pytest.mark.parametrize_device(['native:0', 'cuda:0'])
+@pytest.mark.parametrize('shape,indices', [
+    ((), 0),
+    ((), (1,)),
+    ((), (1, 0)),
+    ((3,), 3),
+    ((3,), (0, 1)),
+    ((2, 3,), (2, 0)),
+])
+def test_getitem_index_error(device, shape, indices):
+    a = array_utils.create_dummy_ndarray(chainerx, shape, 'float32')
+    with pytest.raises(IndexError):
+        out = a[indices]
```

Comment:
CI is not happy about the unused variable it seems.
```suggestion
        a[indices]
```

Reply:
Fixed.